test/modules/header/components/Header.test.js

```diff
@@ -343,18 +346,28 @@ describe('Header', () => {
 		});
 	});
 
-	describe('when modal button is clicked', () => {
-		it('shows a modal window with the showcase', async () => {
+	describe('when routing button is clicked', () => {
+		it('changes the current active tool', async () => {
 			const element = await setup();
 
-			element.shadowRoot.querySelector('.header__modal-button').click();
+			element.shadowRoot.querySelector('.header__routing-button').click();
 
-			expect(store.getState().modal.data.title).toBe('Showcase');
-			//we expect a lit-html TemplateResult as content
-			expect(store.getState().modal.data.content.strings[0]).toBe('<ba-showcase></ba-showcase>');
+			expect(store.getState().tools.current).toBe(Tools.ROUTING);
 		});
 	});
 
+	// describe('when modal button is clicked', () => {
+	// 	it('shows a modal window with the showcase', async () => {
+	// 		const element = await setup();
+
+	// 		element.shadowRoot.querySelector('.header__modal-button').click();
+
+	// 		expect(store.getState().modal.data.title).toBe('Showcase');
+	// 		//we expect a lit-html TemplateResult as content
+	// 		expect(store.getState().modal.data.content.strings[0]).toBe('<ba-showcase></ba-showcase>');
+	// 	});
+	// });
```

Comment:
Let's delete the uncommented lines
https://github.com/ldbv-by/bav4-nomigration/blob/03f67afa7fed3c2ae7282f598fa6c00977f723c9/test/modules/header/components/Header.test.js#L359-L369